cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -333,6 +359,16 @@ func (h eventsRPCHandler) getEvents(request GetEventsRequest) (GetEventsResponse
 			limit = request.Pagination.Limit
 		}
 	}
+	end := events.Cursor{Ledger: request.StartLedger + ledgerbucketwindow.OneDayOfLedgers}
```

Comment:
Is this for limiting the maximum number of ledgers to query? If so it should probably be parameterized in the command-line like we do for transactions or at the very very least it should be commented and use a constant.

Reply:
Also, a day worth of ledgers can be quite a lot. We may need to rethink this based on how expensive it is to query events.

Reply:
We should probably limit by number of events and not ledgers

Reply:
limiting by events still won't solve the issue with current filters. We need to restrict scanning of ledgers for given request. 

> a day worth of ledgers can be quite a lot
hmmm we do look at day of ledgers in memory for events so this might not be true?